apps/darts-modernisation/darts-api/demo.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         CASE_EXPIRY_DELETION_ENABLED: false
         MANUAL_DELETION_ENABLED: false
         PROCESS_E2E_ARM_RPO: false
-        ARM_RPO_HOURS: 24h
+        ARM_RPO_DURATION: 24h
```

Comment:
Its not in a java Duration format e.g. "PT24H", so isn't the name misleading?

Reply:
Spring will auto convert this into a Java Duration on consumption so we do not need the PT prefix here. 

https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.conversion.durations